backend/src/models/api/LevelGetRequest.ts

```diff
@@ -11,6 +12,7 @@ export type LevelGetRequest = Request<
 		emails: EmailInfo[];
 		chatHistory: ChatMessage[];
 		defences: Defence[];
+		chatModel: ChatModel;
```

Comment:
Not that it matters very much, but I assume this chatModel field is actually optional, as we only receive it for Sandbox. I think it would help with clarity to mark it as such.

Reply:
whoops, fixed